### DISCUSSION TITLE
Donation

### DISCUSSION THREAD

**Discussion by TomTasche** [AppIntro/AppIntro#969] [General]
Hi there,

One of my apps is using AppIntro and I'd like to give something back based on the revenue the app generated for me. Therefore I'd like to donate to your project. Would you mind adding a sponsorship-button to GitHub? https://github.com/sponsors

Greetings
Tom

**Comment by cortinico** [comment]
Hey @TomTasche 
First, thank you very much for thinking about us ❤️
I've looked into this and seems like we should be able to get Github Sponsors set up for the AppIntro org in the next days. I'll get back to you once we have it up and running.

**Comment by cortinico** [comment]
Happy to share that now we have a Donate button <3:
https://github.com/sponsors/AppIntro

**Comment by TomTasche** [comment]
Done, keep up the good work 👏

PS: Just a small typo on the sponsor page: "Buying *an* maintaining"
